runtime/doc/syntax.txt

```diff
@@ -5271,51 +5261,10 @@ back to their Vim default.
 Note that if you are using a color scheme, the colors defined by the color
 scheme for syntax highlighting will be lost.
 
-What this actually does is: >
-
-	let g:syntax_cmd = "reset"
-	runtime! syntax/syncolor.vim
-
-Note that this uses the 'runtimepath' option.
-
-							*syncolor*
-If you want to use different colors for syntax highlighting, you can add a Vim
-script file to set these colors.  Put this file in a directory in
-'runtimepath' which comes after $VIMRUNTIME, so that your settings overrule
-the default colors.  This way these colors will be used after the ":syntax
-reset" command.
-
-For Unix you can use the file ~/.config/nvim/after/syntax/syncolor.vim.
```

Comment:
dropping `after/syntax/syncolor.vim` support is intended behavior, but should be mentioned somewhere (like in a vim-diff blurb about the entire thing).

Reply:
I added a new section to `vim-diff.txt`. Let me know if you have any feedback.